packages/store/types/@ember/array/index.d.ts

```diff
@@ -1,7 +1,9 @@
 import EmberArray, { A } from '@ember/array';
+
 namespace EmberArray {
-    function detect(arr: any): boolean;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I think this is maybe a bug with `@typescript-eslint` but I'm not sure how namespaces work.